Userland/Libraries/LibGfx/WebPLoader.cpp

```diff
@@ -68,6 +68,13 @@ struct WebPLoadingContext {
     RefPtr<Gfx::Bitmap> bitmap;
 
     Optional<ReadonlyBytes> icc_data;
+
+    template<size_t N>
+    [[nodiscard]] class Error error(char const (&string_literal)[N])
```

Comment:
What does the `class` in front of the return type do here? (_c++ class return type_ yields no useful results)

Reply:
it disambiguates it from the `Error` enum value.

Reply:
aha, thanks! I feel like that should be an `enum class` perhaps, but that's unrelated :)